src/components/EditableField.jsx

```diff
@@ -80,9 +83,7 @@ const EditableField = (props) => {
         {
           showEdit
             ? (
-              <Tooltip placement='bottom' title='Edit' mouseLeaveDelay={0}>
-                <Button size='small' shape='circle' icon={<EditOutlined />} onClick={toggleEditing} />
-              </Tooltip>
+              <Button size='small' icon={<EditOutlined />} onClick={toggleEditing} />
```

Comment:
is removing the `shape='circle'` keeping the button circular? Why are you removing the tooltip?

Reply:
See about the circle above. Removing the tooltip was a mistake.